### PR TITLE
[UPDATE] install.py

### PR DESCRIPTION
**Pull request by sneekyfoxx**
- Moved `install.py` to **src/**
- Updated **README.md** to include **src/** in link

I tested the script on my system and everything works as intended.

### CODE REVIEW

File: src/install.py
Comment (Sharktheone) [inline src/install.py:1]: I'm not sure if it is really a good idea to put it in an extra directory, since it is only one file.

**Review by Sharktheone — APPROVED** [review]
Looks good, I'll merge your PR as soon as I had time fully testing it.

File: src/install.py
Comment (sneekyfoxx) [reply src/install.py:1]: Feel free to make any changes.

**Comment by Sharktheone** [comment]
I had now time to fully test your changes, it works as expected. Sorry for the long delay and thanks!

**Pull request merged**